configs/microblaze-zephyr-eabi.config

```
@@ -0,0 +1,63 @@
+CT_CONFIG_VERSION="3"
+CT_EXPERIMENTAL=y
+# CT_PREFIX_DIR_RO is not set
+# CT_LOG_PROGRESS_BAR is not set
+CT_ARCH_MICROBLAZE=y
```

set `CT_ARCH_LE=y` to get little endian toolchain.

```suggestion
CT_ARCH_MICROBLAZE=y
CT_ARCH_LE=y
```